### PR TITLE
bug 1772759: fix console operator panic when RouterCAValidationDegraded 

### PR DESCRIPTION
**Pull request by benjaminapetersen**
Reverts router-ca changes in #328.
Not sure yet if this is the direction we want to go, considering https://github.com/openshift/enhancements/pull/126.

I suspect that some of this code can return in a future PR.  We will likely not need the `resource-syncer`, and instead will be able to annotate a new `configmap` with the appropriate `router-ca` annotation to get it injected like the `proxy`. Then we can assess if it exists & mount in the deployment, or not.

/assign @spadgett @jhadvig

### CODE REVIEW

File: pkg/console/starter/starter.go [hunk 1]
```diff
@@ -259,6 +242,9 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 			"Progressing",
 			"Available",
 			// "Faililng"
+			// added in 4.3, can remove in 4.4, to ensure operators running 4.3.x are not wedged.
```
Comment (benjaminapetersen) [inline pkg/console/starter/starter.go]: I'm not sure if this change is strictly necessary.  These were new conditions with the router-ca sync commit within the 4.3 time frame, so if the code is not running in production we can ignore.

Comment (jhadvig) [reply pkg/console/starter/starter.go]: yeah not sure we need this change, if not would rather not add it...

Comment (spadgett) [reply pkg/console/starter/starter.go]: I would take this out, and we can backport the revert to 4.3.

**Comment by spadgett** [comment]
/hold

I'd remove the second commit

File: pkg/console/starter/starter.go [hunk 2]
```diff
@@ -242,6 +242,9 @@ func RunOperator(ctx *controllercmd.ControllerContext) error {
 			"Progressing",
 			"Available",
 			// "Faililng"
```
Comment (spadgett) [inline pkg/console/starter/starter.go]: Not for this PR, but it looks like we can remove these now?

Comment (benjaminapetersen) [reply pkg/console/starter/starter.go]: Yeah we should eliminate.

**Comment by benjaminapetersen** [comment]
Removed the second commit.

**Pull request merged**